api/api.go

```diff
@@ -405,8 +405,16 @@ type CloudBackupGenericRequest struct {
 	// MetadataFilter indicates backups whose metadata has these kv pairs
 	MetadataFilter map[string]string
 	// CloudBackupID must be specified if one needs to enumerate known single
-	// backup( format is clusteruuidORBucketName/srcVolId-SnapId(-incr)
+	// backup (format is clusteruuidORBucketName/srcVolId-SnapId(-incr). If t\
```

Comment:
There is a `t\` at the end of the this comment

Reply:
thanks!. done